controllers/machine_controller.go

```diff
@@ -356,7 +357,7 @@ func (r *MachineReconciler) drainNode(ctx context.Context, cluster *clusterv1.Cl
 		return errors.Errorf("unable to get node %q: %v", nodeName, err)
 	}
 
-	drainer := &kubedrain.Helper{
+	drainer := &drain.Helper{
```

Comment:
Now that `kubectl` is out of tree we no longer need to maintain a copied version of the drain Helper.